### ISSUE TITLE
Debug function is overloaded as a boolean and causing crashes

### ISSUE THREAD

**Issue by jfrumar** [dweinstein/node-google-play#27]
Our service is crashing with this error:

```api_server-v1.0.0.forever.log-TypeError: boolean is not a function
api_server-v1.0.0.forever.log:    at null.<anonymous> (/var/www/groupahead-backend/node_modules/gpapi/lib/api.js:240:13)
api_server-v1.0.0.forever.log-    at wrappedCallback (/var/www/groupahead-backend/node_modules/newrelic/lib/transaction/tracer/index.js:321:23)
api_server-v1.0.0.forever.log-    at wrapped (/var/www/groupahead-backend/node_modules/newrelic/lib/transaction/tracer/index.js:155:28)
api_server-v1.0.0.forever.log-    at wrappedCallback (/var/www/groupahead-backend/node_modules/newrelic/lib/transaction/tracer/index.js:412:66)
api_server-v1.0.0.forever.log-    at wrapped [as _onTimeout] (/var/www/groupahead-backend/node_modules/newrelic/lib/transaction/tracer/index.js:155:28)
api_server-v1.0.0.forever.log-    at Timer.unrefdHandle (timers.js:307:14)```

It looks like the *debug* var both refers to the debug module, and the debug boolean passed to the main function:

https://github.com/dweinstein/node-google-play/blob/master/lib%2Fapi.js#L254

**Comment by dweinstein** [comment]
ah sorry about that. I don't often use the cache option TBH. can get that fixed in a jiffy

**Comment by dweinstein** [comment]
fixed in v3.2.1

**Comment by jfrumar** [comment]
:+1: Nice - thanks for the quick response time. You also saved me from creating the PR
